src/lib/URL.js

```diff
@@ -1,7 +1,7 @@
 /* @flow */
 
 export function isValidURL(link: string): boolean {
-	return /^((https?|ftp):\/\/)?((([a-z\d]([a-z\d-]*[a-z\d])*)\.)+[a-z]{2,}|((\d{1,3}\.){3}\d{1,3}))(:\d+)?(\/[-a-z\d%_.~+]*)*(\?[;&a-z\d%_.~+=-]*)?(#[-a-z\d_]*)?$/i.test(link);
+	return /^((ftp|https?:\/\/)?(\S+\.\S+)+)$/i.test(link);
```

Comment:
It should be `\S+(\.\S+)+` instead of `(\S+\.\S+)+` otherwise for URLs like foo.b.com it will only highlight foo.b.

However I think this is overall a bit too permissive, for example the TLD is guaranteed to be at least 2 characters long and not contain punctuation, etc. This will match, for example, $2.50 as a URL.

I will come up with a better regex, hold on.